app/controllers/landing_page_controller.rb

```diff
@@ -218,7 +213,9 @@ def render_landing_page(default_locale:, locale_param:, structure:)
                          locale_param,
                          topbar_locale)
     marketplace_context = marketplace_context(c, topbar_locale, request)
-    topbar_enabled = feature_enabled?(FEATURE_FLAG)
+
+    FeatureFlagHelper.init(request, false)
+    topbar_enabled = FeatureFlagHelper.feature_enabled?(FEATURE_FLAG)
```

Comment:
Not related to this change but now that I see this I think the FEATURE_FLAG constant was a bad call and makes this less clear than just having the symbol directly here.